this & object prototypes/ch3.md

```diff
@@ -850,15 +857,16 @@ it.next(); // { value:3, done:false }
 it.next(); // { done:true }
 ```
 
-**Note:** We get at the `@@iterator` *internal property* of an object using an ES6 `Symbol`: `Symbol.iterator`. We briefly mentioned `Symbol` semantics earlier in the chapter (see "Computed Property Names"), so the same reasoning applies here. You'll always want to reference such special properties by `Symbol` name reference instead of by the special value it may hold. Also, despite the name's implications, `@@iterator` is **not the iterator object** itself, but a **function that returns** the iterator object -- a subtle but important detail!
+**Nota** Nós chegamos ao `@@iterator` *propriedade interna* de um objeto usando `Symbol` do ES6: `Symbol.iterator`. Nós mencionamos brevemente sobre a semântica do `Symbol` nesse capítulo (veja "Nomes de propriedades computadas"), então o mesmo raciocínio se aplica aqui. Você sempre irá querer referenciar tais propriedades especiais pela referência de nome do `Symbol` em vez de um valor especial. Além disso, apesar das implicações de nomes, `@@iterator` **não é o objeto iterador**, mas uma **função que retorna** o objeto iterador -- um detalhe simples, mas importante!
 
-As the above snippet reveals, the return value from an iterator's `next()` call is an object of the form `{ value: .. , done: .. }`, where `value` is the current iteration value, and `done` is a `boolean` that indicates if there's more to iterate.
+Como o trecho acima revela, o valor de retorno de uma chamada `next()` do iterador é um objeto na forma de `{ value: .. , done: .. }`, onde `value` é o atual valor da iteração e o `done` é um `boolean` que indica se há algo mais para iterar.
 
-Notice the value `3` was returned with a `done:false`, which seems strange at first glance. You have to call the `next()` a fourth time (which the `for..of` loop in the previous snippet automatically does) to get `done:true` and know you're truly done iterating. The reason for this quirk is beyond the scope of what we'll discuss here, but it comes from the semantics of ES6 generator functions.
+Note que o valor `3` foi retornado com `done:false`, que parece estranho à primeira vista. Você tem que chamar o `next()` uma quarta vez (que o laço `for..of` no trecho de código anterior faz automaticamente) para obter `done:true` e saber que você realmente finalizou a iteração. A razão for this quirk estaá aleém do escopo do que iremos discutir aqui, mas vem de semânticas de funções do gerador no ES6.
```

Comment:
A razão ~~for this quirk~~ **deste quirk** estaá ('a' duplicado) aleém ('e' duplicado: `git blame` retornou 'macOs' haha)